File: app/src/main/java/cn/surine/schedulex/data/entity/CourseList.java
package cn.surine.schedulex.data.entity;

import java.util.List;

/**
 * Intro：
 *
 * @author sunliwei
 * @date 2020-01-16 20:42
 */
public class CourseList extends BaseVm {
    //当前周
    public int nowWeek;
    //当周课表
    public List<Course> courseList;
    //共多少周
    public int weeks;
}

File: app/src/main/java/cn/surine/schedulex/data/entity/CourseList.kt
package cn.surine.schedulex.data.entity

/**
 * Intro：
 *
 * @author sunliwei
 * @date 2020-01-16 20:42
 */
class CourseList : BaseVm() {
    //当前周
    @JvmField
    var nowWeek = 0
    //当周课表
    @JvmField
    var courseList: List<Course>? = null
    //共多少周
    @JvmField
    var weeks = 0
}